apps/src/blockly/addons/cdoFieldDropdown.js

```diff
@@ -149,6 +149,33 @@ export default class CdoFieldDropdown extends GoogleBlockly.FieldDropdown {
     super.toXml(element);
     return element;
   }
+
+  /**
+   * Override of createTextArrow_ to fix the arrow position on Safari.
+   * We need to add dominant-baseline="central" to the arrow element in order to
+   * center it on Safari.
+   *  @override */
+  createTextArrow_() {
+    // TODO: This field changes from arrow_ to arrow with the v10 upgrade.
```

Comment:
This would likely break the field on upgrade right?
Would adding something like `this.arrow = this.arrow_;` make it forwards compatible? I see that the property is referenced in a few other methods of the base class (`applyColour` and `renderSelectedImage`).

Reply:
good idea, I'll update this!

Reply:
Great!